tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -529,6 +530,79 @@ public async Task CertificateOverrideAsync()
             }
         }
 
+        [TestMethod]
+        public async Task CertificateOverrideAsync()
```

Comment:
This seems to be an old method, should be removed.
Also cert on line 482 above is not actually added to the app.